tests/foreman/destructive/test_infoblox.py

```diff
@@ -20,17 +20,11 @@
 
 pytestmark = pytest.mark.destructive
 
-dhcp_isc_package = 'rubygem-smart_proxy_dhcp_remote_isc'
 infoblox_package = 'rubygem-infoblox'
 infoblox_dhcp_package = 'rubygem-smart_proxy_dhcp_infoblox'
 infoblox_dns_package = 'rubygem-smart_proxy_dns_infoblox'
 
 params = [
-    (
-        'enable-foreman-proxy-plugin-dhcp-remote-isc',
-        {'foreman-proxy-dhcp': 'true'},
-        f'rpm -q {dhcp_isc_package}',
```

Comment:
why removing isc_dhcp? could you elaborate how this is failing for IPv6?

Reply:
As I mentioned in the description that as we don't really test isc_dhcp with infoblox integration and we don't document that as well. SO there's not pointing keeping it for infoblox tests